src/erl_betree.erl

```diff
@@ -36,3 +40,12 @@ check_clock_type(?CLOCK_REALTIME) -> ?CLOCK_REALTIME;
 check_clock_type(?CLOCK_PROCESS_CPUTIME_ID) -> ?CLOCK_PROCESS_CPUTIME_ID;
 check_clock_type(?CLOCK_THREAD_CPUTIME_ID) -> ?CLOCK_THREAD_CPUTIME_ID;
 check_clock_type(_) -> ?CLOCK_MONOTONIC.
+
+search_iterator(Betree, Event) ->
+    erl_betree_nif:search_iterator(Betree, Event).
+search_next(Iterator) ->
+    erl_betree_nif:search_next(Iterator).
+search_all(Iterator) ->
+    erl_betree_nif:search_all(Iterator).
+search_iterator_release(Iterator) ->
```

Comment:
If the owner process dies, does this automatically release the created search iterators?

Reply:
Yes, it does.
MEM_SEARCH_ITERATOR resource has destructor, https://github.com/adgear/erl-be-tree/blob/PEDSP-2941/c_src/betree.c#L205,
https://github.com/adgear/erl-be-tree/blob/PEDSP-2941/c_src/betree.c#L152